src/bin/sol-fbp-to-dot/main.c

```diff
@@ -337,7 +340,8 @@ main(int argc, char *argv[])
 
     if (convert_fbp_to_dot(&input_graph, dotfile)) {
```

Comment:
I would remove the brackets here and change the if condition to

if (convert_fbp_to_dot(&input_graph, dotfile) != 0)
   r = EXIT_SUCCESS;
else
    fprintf(stderr, "Couldn't convert from fbp to ...